slidedeck/src/slides/03-react-state/05-component-lifecycle.pug

```diff
@@ -0,0 +1,28 @@
+section
```

Comment:
I think that we might need more content/examples around lifecycle methods. Might be its own section?